gax/test/test-application/src/index.ts

```diff
@@ -417,14 +422,17 @@ async function testEchoErrorWithRetries(client: SequenceServiceClient) {
     await client.attemptSequence(attemptRequest, settings);
   } catch (err) {
     assert.strictEqual(JSON.stringify((err as GoogleError).code), '4');
-    assert.match(
-      JSON.stringify((err as GoogleError).message),
-      /Exceeded maximum number of retries retrying error Error: 14 UNAVAILABLE: 14 before any response was received/,
-    );
+    assert.match((err as GoogleError).message, /Exceeded maximum number of retries retrying error Error: 4 DEADLINE_EXCEEDED: Deadline exceeded after/);
+    assert.match((err as GoogleError).message, /waiting for metadata filters before any response was received : Previous errors : \[{message: 14 UNAVAILABLE: 14, code: 14, details: , note: },{message: 4 DEADLINE_EXCEEDED: Deadline exceeded after/);
```

Comment:
See my comment below - we should not be matching on this message - this `DEADLINE_EXCEEDED` should be coming from https://github.com/googleapis/gax-nodejs/blob/bb11949f04ef70314c4b0d9b542e81822d59f352/gax/src/normalCalls/retries.ts#L133-L138

Reply:
It looks like the fix upstream means the test could be working correctly so I'll fix the test.